ZenPacks/zenoss/OpenStackInfrastructure/datasources/AMQPDataSource.py

```diff
@@ -168,7 +167,7 @@ def onError(self, result, config):
             'summary': errmsg,
             'severity': ZenEventClasses.Error,
             'eventKey': 'openstackCeilometerAMQPCollection',
```

Comment:
I think eventKey is now redundant, since the eventClassKey will be sufficient for deduplication.